Learn-CSS-Blog/index29.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav>
+                <ul class="nav-links">
+                    <li><a href="#">Intro</a></li>
+                    <li><a href="#">Blog</a></li>
+                </ul>
+                <a href="#" id="logo" >
+                    <img src="https://d33wubrfki0l68.cloudfront.net/e66392bc2941ad564b6956321e778f29073cf585/4190b/img/glyph-alternative.svg" alt="freeCodeCamp logo">
+                </a>
+            </nav>
+        </header>
+        <section class="container">
+            <div class="about">
+                <div>
+                    <img class="img" src="img/profile.jpg" alt="profile picture">
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Add a second `div' element after the closing `</div>` tag. 
```

Comment:
Since this test description will eventually move up to the area where the learner will write code, maybe we could simplify this a bit:
```
Add a second `<div>' element to the "about" `<div>`. 
```